lgc/patch/PatchEntryPointMutate.cpp

```diff
@@ -965,9 +965,12 @@ uint64_t PatchEntryPointMutate::generateEntryPointArgTys(ShaderInputs *shaderInp
       *userDataArg.argIndex = argTys.size() + argOffset;
     unsigned dwordSize = userDataArg.argDwordSize;
     if (userDataArg.userDataValue != static_cast<unsigned>(UserDataMapping::Invalid)) {
+      // Most of user data metadata entries is 1 except root push descriptors
+      bool systemUserData = userDataArg.userDataValue >= static_cast<unsigned>(UserDataMapping::GlobalTable);
```

Comment:
nit: The usual convention is to prefix boolean variables with a verb, e.g., `hasSystemUserData`, `isSystemUserData`, `containsSystemUserData`. Could you prefix this variable name with whatever fits best?